src/test/java/com/stripe/functional/GeneratedExamples.java

```diff
@@ -3002,4 +3002,62 @@ public void testSetupIntentVerifyMicrodeposits() throws StripeException {
         "/v1/setup_intents/seti_xxxxxxxxxxxxx/verify_microdeposits",
         params.toMap());
   }
+
+  @Test
+  public void testTestClockCreate() throws StripeException {
+    com.stripe.param.testhelpers.TestClockCreateParams params =
```

Comment:
I wonder why it's using the fully qualified name here? This is technically correct though.

Reply:
Hmm, I think the test generator declared bankruptcy on namespaced params altogether as a solution for being unable to import both `com.stripe.param.billingportal.SessionCreateParams` and `com.stripe.param.checkout.SessionCreateParams`.